docs/guides/software-setup/software-windows.md

```diff
@@ -0,0 +1,353 @@
+# Local Software Setup - Windows
+
+## Overview
+
+- **This guide assumes you are on Windows.** [Mac](software-setup/mac.md) and [Linux](software-setup/linux.md) have different methods.
+- **Please read every step of this guide carefully.** It is highly recommend doing a full read-through before you even get started.
+
+Within this guide, we will be installing the following software:
+
+- [XAMPP](https://www.apachefriends.org/download.html) for the local webserver and database
+- [SourceGit](https://sourcegit-scm.github.io) for our graphical git client
+
+!!! info "Windows Firewall Pop-ups"
+    At any point in time while installing the below software, you may see a pop-up like this. **Click "Allow" to allow the software to finish installing.** All of the recommended software is safe.
+
+    <figure markdown="span">
+    ![Windows Firewall pop-up](../../images/software-setup/windows/xampp-firewall.png){ width="600" }
+    </figure>
+
+## XAMPP Setup
+
+### Installing XAMPP
+
+1. Download [XAMPP](https://www.apachefriends.org/download.html).
+
+  - For LK v2, select **PHP 8.1**.
```

Comment:
Given these are intended to be the docs for v3 does it really make sense to still list this here? 🤔

Reply:
Oh, missed that one-- but 3.0/Laravel 10 also requires 8.1 minimum, so the version itself is fine.